src/lib.rs

```diff
@@ -13,7 +15,7 @@ pub use error::{Error, Result};
 pub use map::{Map, MapBuilder};
 pub use stream::{IntoStreamer, Streamer};
 
-mod regex;
+pub mod regex;
```

Comment:
I think you added that to fix a doctest. This is not needed. The problem is fixed in master